pkgs/by-name/gi/gitlab-release-cli/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitLab
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "gitlab-release-cli";
+  version = "0.18.0";
+
+  src = fetchFromGitLab {
+    owner = "gitlab-org";
+    repo = "release-cli";
+    rev = "v${version}";
+    hash = "sha256-CCSice/uMf2OfFNEpwwhX6A0wrSsC1v9XWEhAAwQRso=";
+  };
+
+  vendorHash = "sha256-UwDMRsWbk8rEv2d5FssIzCLby68YZULoxd3/JGLsCQU=";
+
```

```suggestion

doCheck = !stdenv.isDarwin;
```

Not sure why they are failing on darwin though...

---

Looking at the build output I see

>  Error:          "failed to create release: failed to do request: Post \"https://127.0.0.1:56138/api/v4/projects/projectID/releases\": tls: failed to verify certificate: x509: “127.0.0.1” certificate is not standards compliant" does not contain "x509: certificate signed by unknown authority"

Are the checks dependent on external services that are not running in a NixOS build?
Or is the `$NIX_SSL_CERT_FILE` possibly not found/used?

---

How about skipping the failing test individually until the underlying cause of the test failure on darwin is found? Note that if this suggestion is accepted `stdenv` needs to be added to the inputs (see suggestion above).
```suggestion
  checkFlags = lib.optionals stdenv.isDarwin [
    # Skip failing test
    "-skip TestHTTPSCustomCA"
  ];
```